app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(human["name"], human["age"]) for human in people]
+
+    for spouse in people:
+        person = Person.people[spouse["name"]]
+
+        if wife := spouse.get("wife"):
+            person.wife = Person.people[wife]
+        elif husband := spouse.get("husband"):
+            person.husband = Person.people[husband]
```

Comment:
Suggestion: You can avoid nested 'if' statements by using logical operators 'or' and 'and'. For example, you can combine the conditions for 'wife' and 'husband' using 'or'.